python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -472,6 +635,12 @@ def transpile(self,
                 recv_op_role_var_name = splited_trainer_grad[0].name
 
             if param_varname in self.sparse_param_to_height_sections:
+
+                for table_name in table_names:
+                    distributed_var = self.vars_overview.get_distributed_var_by_slice(
```

Comment:
It's a good idea to pack variable distribution but seems `vars_overview` is only used for remote table, this can be used by the whole transpiler though. maybe I can try make the transpiler implementation simpler using this class.

Reply:
```vars_overview`` contains all variables on the parameter server, not only ```vars_overview ```, I will use it in io.py when ```save_persistables``` and ```load_persistables```.